src/branding/aosc/Makefile.am

```diff
@@ -0,0 +1,10 @@
+aoscbrandingdir = $(datadir)/cockpit/branding/aosc
+
+dist_aoscbranding_DATA = \
+	src/branding/aosc/branding.css \
+	$(NULL)
+
+install-data-hook::
+	ln -sTfr $(DESTDIR)/usr/share/cockpit/aosc-branding/backdrop.jpg $(DESTDIR)$(aoscbrandingdir)/backdrop.jpg
+	ln -sTfr $(DESTDIR)/usr/share/cockpit/aosc-branding/apple-touch-icon.png $(DESTDIR)$(aoscbrandingdir)/apple-touch-icon.png
+	ln -sTfr $(DESTDIR)/usr/share/cockpit/aosc-branding/favicon.ico $(DESTDIR)$(aoscbrandingdir)/favicon.ico
```

Comment:
These are weird symlink targets -- these are supposed to point to some existing system resources, such as /usr/share/pixmaps/ or so -- files that are delivered by the operating system. Where would they come from?

Reply:
Done